sda/cmd/api/api.md

```diff
@@ -88,3 +88,33 @@ Admin endpoints are only available to a set of whitelisted users specified in th
     ```bash
     curl -H "Authorization: Bearer $token" -X POST  https://HOSTNAME/dataset/release/my-dataset-01
     ```
+
+- `/users`
+  - accepts `GET` requests`
+  - Returns all users with active uploads as a JSON array
+
+    Example:
+
+    ```bash
+    curl -H "Authorization: Bearer $token" -X GET  https://HOSTNAME/users
+    ```
+
+- Error codes
+  - `200` Query execute ok.
+  - `401` User is not in the list of admins.
+  - `500` Internal error due to DB failure.
+
+- `/users/:username/files`
+  - accepts `GET` requests`
+  - Returns all files for a user with active uploads as a JSON array
+
+    Example:
+
+    ```bash
+    curl -H "Authorization: Bearer $token" -X GET  https://HOSTNAME/users
+    ```
+
+- Error codes
+  - `200` Query execute ok.
+  - `401` User is not in the list of admins.
```

Comment:
There's a potential for confusion here; that `User` could be interpreted as the user in `/user/:username/file` (instead of the requester)

Reply:
so `:username` should be `:submitter` to make things more clear.

Reply:
Reworded the readme